app/helpers/heatmap_helper.rb

```diff
@@ -493,9 +500,10 @@ def self.top_n_query(query, sort_by, num_results, threshold_filters)
           count_type = '#{sort[:count_type]}' DESC,
           #{sort[:metric]} #{sort[:direction] == 'highest' ? 'DESC' : 'ASC'}
       ) b
-      -- Overfetch by a factor of 4 to allow for a) both count types, and
+      -- Overfetch by a factor of 8 to allow for a) both count types, and
```

Comment:
We do not want to change the factor unless the feature is enable.
I am not sure about the factor 8 also, because you will probably get a lot more species than genus... 
Any concerns with that?

Reply:
Reverted back to 4!